tools/build/make.py

```diff
@@ -205,13 +213,12 @@ def default_cross_toolchain():
         # if not os.getenv("X_COMPILER_TYPE"):
         #    new_env_vars["X_COMPILER_TYPE"] = parsed_args.cross_compiler_type
 
+    libbsd_install_dir = bootstrap_libbsd(source_root, objdir_prefix)
```

Comment:
Bootstrapping libbsd should only be done `if sys.platform.startswith("linux"):` since it is not needed for building on MacOS.